pipeline/filters/kubernetes.md

```diff
@@ -76,7 +81,18 @@ To perform processing of the _log_ key, it's **mandatory to enable** the _Merge\
 
 If _log_ value processing fails, the value is untouched. The order above is not chained, meaning it's exclusive and the filter will try only one of the options above, **not** all of them.
 
-## Kubernetes Annotations
+## Kubernetes Namespace Meta
+Namespace Meta can be enabled via the following settings: 
+* Namespace\_Labels
+* Namespace\_Annotations
+
+Using any Namespace Meta requires the use of the Kube API as it can not be fetched directly from Kubelet. If `Use_Kubelet On` has been set, the Kubelet api will only be used to fetch pod metadata, while namespace meta is fetched from the upstream Kubernetes API.
```

Comment:
@ryanohnemus could you make sure to split sentences on to single lines for docs? It makes it easier to review and comment on because you can only comment on a line.